eq-author-api/schema/resolvers/base.js

```diff
@@ -501,6 +489,10 @@ const Resolvers = {
       questionnaire.shortTitle || questionnaire.title,
   },
 
+  User: {
+    displayName: user => user.name || user.email,
```

Comment:
Is this still necessary?

Reply:
Yeah, this doesn't fix the issue with not sending the name on login, it just stops us saving the email against the questionnaire so if they login a second time it fixes itself.